docs/api-reference/front-commerce-remix/graphql-playground.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "GraphQL Playground"
+sidebar_position: 2
+description:
+  "The Graphql Playground allows you to expose a Graphql Yoga playground in your
+  application."
+---
+
+<p>{frontMatter.description}</p>
+
+## Installation
+
+First you need to create a new route under `/routes/graphql.ts`:
+
+Then you can simply re-export the loader and action from the
+`@front-commerce/remix` package
+
+```ts title="/app/routes/graphql.ts"
+export { loader, action } from "@front-commerce/remix/playground.server";
+```
+
+:::caution
+
+The playground is currently only available on the `/graphql` route, if you add
+it to any other routes, it will try to redirect to the `/graphql` route.
+
+:::
+
+## Playground in Production
+
+To enable it in production you need to set the `FRONT_COMMERCE_ENV` environment
+variable to `production`
```

Comment:
**issue:** this isn't correct. We don't allow it in production, and don't recommend it.

**suggestion:** 
```suggestion
## Playground in a staging environment

The playground must not be enabled in a production environment. However, it is useful in staging environments. To enable it in a production build you need to set the `FRONT_COMMERCE_ENV` environment
variable to `dev`. The application will start as usual, but you could access the playground to query your live GraphQL server.
```